tests/acceptance/features/webUISharingPublicBasic/publicLinkCreate.feature

```diff
@@ -172,10 +172,10 @@ Feature: Create public link shares
       | uid_owner   | Alice          |
       | permissions | read           |
       | path        | /simple-folder |
-      | name        | Quicklink      |
+      | name        | Link           |
     And the following success message should be displayed on the webUI
```

Comment:
due to this change the test is failing with latest ocis https://github.com/owncloud/web/issues/8929. The then step asserts from an api call. So if I understand this correctly. The naming from `QuickLink` to `link` is just on the UI level and API stills responds with `Quicklink`?

Reply:
Yes, that was just a UI change.